node/cn/metrics.go

```diff
@@ -21,51 +21,56 @@
+	propBlockInTrafficMeter      = metrics.NewRegisteredMeter("klay/prop/blocks/in/traffic", nil)
+	propBlockOutPacketsMeter     = metrics.NewRegisteredMeter("klay/prop/blocks/out/packets", nil)
+	propBlockOutTrafficMeter     = metrics.NewRegisteredMeter("klay/prop/blocks/out/traffic", nil)
+	reqHeaderInPacketsMeter      = metrics.NewRegisteredMeter("klay/req/headers/in/packets", nil)
+	reqHeaderInTrafficMeter      = metrics.NewRegisteredMeter("klay/req/headers/in/traffic", nil)
+	reqHeaderOutPacketsMeter     = metrics.NewRegisteredMeter("klay/req/headers/out/packets", nil)
+	reqHeaderOutTrafficMeter     = metrics.NewRegisteredMeter("klay/req/headers/out/traffic", nil)
+	reqBodyInPacketsMeter        = metrics.NewRegisteredMeter("klay/req/bodies/in/packets", nil)
+	reqBodyInTrafficMeter        = metrics.NewRegisteredMeter("klay/req/bodies/in/traffic", nil)
+	reqBodyOutPacketsMeter       = metrics.NewRegisteredMeter("klay/req/bodies/out/packets", nil)
+	reqBodyOutTrafficMeter       = metrics.NewRegisteredMeter("klay/req/bodies/out/traffic", nil)
+	reqStateInPacketsMeter       = metrics.NewRegisteredMeter("klay/req/states/in/packets", nil)
+	reqStateInTrafficMeter       = metrics.NewRegisteredMeter("klay/req/states/in/traffic", nil)
+	reqStateOutPacketsMeter      = metrics.NewRegisteredMeter("klay/req/states/out/packets", nil)
+	reqStateOutTrafficMeter      = metrics.NewRegisteredMeter("klay/req/states/out/traffic", nil)
+	reqReceiptInPacketsMeter     = metrics.NewRegisteredMeter("klay/req/receipts/in/packets", nil)
+	reqReceiptInTrafficMeter     = metrics.NewRegisteredMeter("klay/req/receipts/in/traffic", nil)
+	reqReceiptOutPacketsMeter    = metrics.NewRegisteredMeter("klay/req/receipts/out/packets", nil)
+	reqReceiptOutTrafficMeter    = metrics.NewRegisteredMeter("klay/req/receipts/out/traffic", nil)
+	miscInPacketsMeter           = metrics.NewRegisteredMeter("klay/misc/in/packets", nil)
+	miscInTrafficMeter           = metrics.NewRegisteredMeter("klay/misc/in/traffic", nil)
+	miscOutPacketsMeter          = metrics.NewRegisteredMeter("klay/misc/out/packets", nil)
+	miscOutTrafficMeter          = metrics.NewRegisteredMeter("klay/misc/out/traffic", nil)
+	txReceiveCounter             = metrics.NewRegisteredCounter("klay/tx/recv/counter", nil)
+	txResendCounter              = metrics.NewRegisteredCounter("klay/tx/resend/counter", nil)
+	txSendCounter                = metrics.NewRegisteredCounter("klay/tx/send/counter", nil)
+	txResendRoutineGauge         = metrics.NewRegisteredGauge("klay/tx/resend/routine/gauge", nil)
+	cnPeerCountGauge             = metrics.NewRegisteredGauge("p2p/CNPeerCountGauge", nil)
+	pnPeerCountGauge             = metrics.NewRegisteredGauge("p2p/PNPeerCountGauge", nil)
+	enPeerCountGauge             = metrics.NewRegisteredGauge("p2p/ENPeerCountGauge", nil)
+	propConsensusInPacketsMeter  = metrics.NewRegisteredMeter("klay/prop/consensus/in/packets", nil)
```

Comment:
How about this? This packet is istanbulMsg. So I suggest like this.

```suggestion
	propConsensusIstanbulInPacketsMeter  = metrics.NewRegisteredMeter("klay/prop/consensus/istanbul/in/packets", nil)
```

Reply:
@KimKyungup Got it. The metrics only works for istanbul consensus, it seems better.